docs/src/main/paradox/scala/http/routing-dsl/rejections.md

```diff
@@ -33,7 +33,7 @@ the @ref[handleRejections](directives/execution-directives/handleRejections.md#h
 ## The RejectionHandler
 
 The @ref[handleRejections](directives/execution-directives/handleRejections.md#handlerejections) directive delegates the actual job of converting a list of rejections to its argument, a
-@github[RejectionHandler](/akka-http/src/main/scala/akka/http/scaladsl/server/RejectionHandler.scala), which is defined like this:
```

Comment:
Hmm... Links to sources have been kind of historically spray style and perhaps not the best idea for us in furniture esp with java docs etc. So I think I'm in favour of this change of linking to docs instead, thanks :)